client_apis/python/src/cbapi/cbapi.py

```diff
@@ -406,6 +407,80 @@ def watchlist_modify(self, id, watchlist):
 
         return r.json()
 
+    def watchlist_action_get(self, watchlist_id):
+        '''
```

Comment:
FYI, you're being inconsistent w/ using single quote vs. double-quote for docstrings.  Your other new methods use double-quote, I say to use that here as well